posthog/hogql_queries/utils/formula_ast.py

```diff
@@ -0,0 +1,67 @@
+import ast
+import operator
+from typing import Any, Dict, List
+
+
+class FormulaAST:
```

Comment:
This deserves a small test suite of its own, so I propose adding formula support in separate PR